### PR TITLE
Refactor/scripts

### PR DESCRIPTION
**Pull request by anewb87**
#### What's this PR do?
-Refactor scripts.js
-Changed home button (deleted home button and made the 'what's cookin' words/logo a home 'button')
-Added a missing test

#### Where should the reviewer start?
scripts.js, Recipe-Repository-test.js

#### How should this be manually tested?
Just check the scripts file for the refactoring and the test file for the additional test
Open index.html and check out the button

#### Any background context you want to provide?
These were things that Hannah suggested we fix.

#### What are the relevant tickets?
Na

#### Screenshots (if appropriate)
Na

#### Questions:
Nope

### CODE REVIEW

**Review by cagallo — APPROVED** [review]
Great refactor! Cleaned up scripts a ton! great job!

File: src/scripts.js [hunk 1]
```diff
@@ -36,33 +36,39 @@ const filterSection = document.querySelector('.filter-section-js');
...
 const cookbookPageButton = document.querySelector('.cookbook-page-button-js');
 const singleViewFavoriteButton = document.querySelector('.single-view-favorite-button-js');
-const pageTitleButton = document.querySelector('.page-title-js');
+const pageTitle = document.querySelector('.page-title-js');
 
 // EVENT LISTENERS
 searchRecipesButton.addEventListener('click', () => {
   searchAllRecipes(recipeRepository.recipeData);
 });
+searchBar.addEventListener('keyup', function(e) {
```
Comment (aliroemhildt) [inline src/scripts.js:52]: I'm glad we could add this feature. A small change that makes it way more user friendly!

**Review by aliroemhildt — APPROVED** [review]
All of these changes look great and really cleans things up!

**Review by kammon10 — APPROVED** [review]
nice work!

**Pull request merged**